### PR TITLE
Update the doc for Silverpeas

### PR DESCRIPTION
**Pull request by mmoqui**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.idea
```
Comment (yosifkit) [inline .gitignore]: I don't think this file is necessary.

Comment (mmoqui) [inline .gitignore]: This file is just for informing to git to ignore some IDE specific files (like here the .idea directory created by IntelliJ IDEA)

Comment (tianon) [inline .gitignore]: Please use local ignores instead (`.git/info/exclude`; see https://help.github.com/articles/ignoring-files/#explicit-repository-excludes).

Comment (mmoqui) [inline .gitignore]: done

**Pull request merged**